### PR TITLE
add tabby pills to twitter feed section as described in #115

### PR DESCRIPTION
**Pull request by aeschylus**
#115

### CODE REVIEW

**Comment by aeschylus** [comment]
Tests and such are failing due to not having a unique key name. So this is pending the renaming of twitter accounts, which should happen tomorrow.

**Comment by camillevilla** [comment]
✨
twitter_tabs

File: src/components/Dashboard.jsx [hunk 1]
```diff
@@ -141,31 +145,25 @@ class Dashboard extends React.Component {
...
-            return (
-              <StatusItem
-                key={endpointName}
-                serviceName={endpoint.displayName}
-                serviceStatus={endpoint.status}
-                statusMessage={statuses[endpoint.status].message}
-                statusIcon={statuses[endpoint.status].icon}
-              />
-            );
+             // Return the element. Also pass key
```
Comment (mejackreed) [inline src/components/Dashboard.jsx]: I'm seeing some extra indention here that may not be needed. Could this be looked into? Probably an editor gone awry  🤖

Comment (camillevilla) [reply src/components/Dashboard.jsx]: Er how's this @mejackreed?

**Pull request merged**